docs/options.md

```diff
@@ -90,20 +90,31 @@ Default: `auto`
 
 `auto` will auto-detect platform using environment variables, such as `TRAVIS_OS_NAME`/`APPVEYOR`/`CIRCLECI`.
 
-For `linux` you need Docker running, on macOS or Linux. For `macos`, you need a Mac machine, and note that this script is going to automatically install MacPython on your system, so don't run on your development machine. For `windows`, you need to run in Windows, and `cibuildwheel` will install required versions of Python to `C:\cibw\python` using NuGet.
+- For `linux`, you need Docker running, on macOS or Linux.
+- For `macos`, you need a Mac machine. Note that cibuildwheel is going to install MacPython on your system, so you probably don't want to run this on your development machine.
+- For `windows`, you need to run in Windows. cibuildwheel will install required versions of Python to `C:\cibw\python` using NuGet.
```

Comment:
I didn't realize this was scoped like this, nice. :) So it's really only macOS that gets true "global" installs.

Reply:
Yeah. We could even install the Pythons somewhere nice from [appdirs], then maybe we could drop the 'only on CI' warning. 
[appdirs]: https://pypi.org/project/appdirs/

On Mac, I seem to remember that you couldn't install it to a different location for some reason. The `installer` command supports it, but something in Python wasn't happy.